compiler/qsc_partial_eval/tests/qubits.rs

```diff
@@ -36,20 +38,42 @@ fn qubit_ids_are_correct_for_allocate_use_release_one_qubit() {
         }
         "#,
     });
-    let op_callable_id = CallableId(1);
-    assert_callable(&program, op_callable_id, &single_qubit_intrinsic_op());
-    assert_block_instructions(
-        &program,
-        BlockId(0),
-        &[
-            Instruction::Call(
-                op_callable_id,
-                vec![Operand::Literal(Literal::Qubit(0))],
-                None,
-            ),
-            Instruction::Return,
-        ],
-    );
+    expect![[r#"
```

Comment:
Would it make sense to update all the partial eval tests to expect-macro patterns? In making this change, I found I'd have to manually update almost every test that generates RIR and checks the value against a constructed program. If the tests were string based like this one example shows, then a global expect-update would add the needed strings everywhere. It would also remove the need for the test utilities `assert_callable` and `assert_block_instructions` since all that same info (and more) is present in the strings.

Reply:
Also, until the rest of the tests are updated, the CI build will fail, so we should decide one way or the other either as part of this PR or in a separate PR what pattern we'd like to follow.

Reply:
Trying out a targeted expect-test based approach, let me know what you think.